exchanges/account/account_test.go

```diff
@@ -410,3 +410,87 @@ func TestSave(t *testing.T) {
 	assert.Equal(t, 80.0, e.total)
 	assert.Equal(t, 20.0, e.hold)
 }
+
+func TestUpdateBalance(t *testing.T) {
```

Comment:
```suggestion
func TestUpdate(t *testing.T) {
```
To match the function name you are testing